release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -192,6 +192,22 @@ installation does not fail {product-title} cluster installation.
 You can xref:../openshift_images/samples-operator-alt-registry.adoc#installation-restricted-network-samples_samples-operator-alt-registry[configure alternate or mirrored registries]
 to bypass this issue if it arises during your cluster installation.
 
+[id="ocp-4-4-developer-experience"]
```

Comment:
Would you mind moving category to after the Scale category? Our RN writer has another PR open that was also adding it and that's where he had it placed.